.travis.yml

```diff
@@ -14,6 +12,7 @@ before_install:
   - composer self-update
 
 install:
-  - composer install
+  - if [[ $TRAVIS_PHP_VERSION != '5.5' ]]; then composer install ; fi
+  - if [[ $TRAVIS_PHP_VERSION == '5.5' ]]; then composer update --prefer-lowest ; fi
```

Comment:
IMO, the lowest job should be a dedicated job, also running PHP 5.5 with uptodate deps